decidim-core/app/views/layouts/decidim/_wrapper.html.erb

```diff
@@ -82,7 +82,9 @@ end
                 </div>
               </div>
             <% end %>
-            <%= render partial: "layouts/decidim/edit_link" %>
+
+            <%= content_for(:extra_admin_link) if content_for?(:extra_admin_link) %>
+            <%= content_for(:edit_link) if content_for?(:edit_link) %>
```

Comment:
I know that is no big deal, but I'd keep the separate template for these links. Instead of removing it, I would just rename it as `admin_links`. 
When using the Decidim gem, it is very useful to have the hability to override small parts of the code, so many small templates are better than one big template.

Reply:
I understand your point, ok I'll add these two `content_for` in a separate partial as suggested `_admin_links.html.erb`. 😃